lib/std/c/darwin.zig

```diff
@@ -1007,7 +1007,16 @@ pub const sockaddr = extern struct {
     data: [14]u8,
 
     pub const SS_MAXSIZE = 128;
-    pub const storage = std.x.os.Socket.Address.Native.Storage;
+    pub const storage = extern struct {
```

Comment:
This branch does not introduce the "storage" declaration; it only fixes it. Since the declaration is used in some places, efforts to remove it should be a follow-up changeset to this.

```
$ grep -RI sockaddr.storage ../lib/std/
../lib/std/c/solaris.zig:    addr: sockaddr.storage,
../lib/std/c/solaris.zig:        addr: sockaddr.storage,
../lib/std/c/solaris.zig:        dstaddr: sockaddr.storage,
../lib/std/c/solaris.zig:        broadaddr: sockaddr.storage,
../lib/std/c/solaris.zig:        token: sockaddr.storage,
../lib/std/c/solaris.zig:        subnet: sockaddr.storage,
../lib/std/c/freebsd.zig:            address: sockaddr.storage,
../lib/std/c/freebsd.zig:            peer: sockaddr.storage,
```